src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -874,6 +933,20 @@ export class AppRoot extends mixinBehaviors
     this.maybeCloseDrawer();
   }
 
+  _sortLanguageNameAscending(a, b) {
```

Comment:
Sort when you set the language list. It's easier to understand and doesn't rely on Polymer (and maybe one may want to use a different order).

Reply:
Done